tests/data/app/view/form/button_in_link.php

```diff
@@ -0,0 +1,10 @@
+<html>
+<body>
+<form action="/form/button" method="POST">
+    <button type="submit" name="btn0">Submit</button>
+</form>
+<a href="/info"><input type="submit" value="More Info"></a>
+
+<a href="/info"><span><input type="submit" value="Span Info"></span></a>
```

Comment:
from the point of valid HTML markup I'd say this is a dirty hack :/
not sure how many people actually use it (but they shouldn't)